src/relayserver/ServerConfigParams.ts

```diff
@@ -79,9 +81,11 @@ const serverDefaultConfiguration: ServerConfigParams = {
   alertedBlockDelay: 0,
   minAlertedDelayMS: 0,
   maxAlertedDelayMS: 0,
-  maxAcceptanceBudget: 2e5,
+  // set to paymasters' default acceptanceBudget + RelayHub.calldataGasCost(<paymasters' default calldataSizeLimit>)
+  maxRelayExposure: defaultPaymasterConfiguration.acceptanceBudget + defaultRelayHubConfiguration.dataGasCostPerByte * defaultPaymasterConfiguration.calldataSizeLimit,
```

Comment:
why? the whole idea is have a number here, e.g. 300k

Our tests then backtrack and make sure this number is viable.

Reply:
I don't necessarily agree that it should be detached from how we got to that number. This default should be in line with the paymasters' and hubs' constants.